### PR TITLE
Chore: Remove extra space in blogpost template

### PR DESCRIPTION
**Pull request by kaicataldo**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[x] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

**What changes did you make? (Give an overview)**
Remove an extra space character in the blogpost template. It doesn't actually get rendered on the website, but does exist in markdown files.

Example: https://github.com/eslint/eslint.github.io/commit/e5576a225da2dcbe2182c9e581be9c8e62e62d7a#diff-4a34709f3e425809d29bd54136cac633R10

**Is there anything you'd like reviewers to focus on?**
Nothing in particular

### CODE REVIEW

File: templates/blogpost.md.ejs [hunk 1]
```diff
@@ -7,7 +7,7 @@ tags:
 ---
 # ESLint v<%= version %> released
 
-We just pushed ESLint v<%- version %>, which is a <%- type %> release upgrade of ESLint. This release <% if (type !== "patch") { %>adds some new features and<% } %> fixes several bugs found in the previous release. <% if (type === "major") { %>This release also has some breaking changes, so please read the following closely. <% } %>
+We just pushed ESLint v<%- version %>, which is a <%- type %> release upgrade of ESLint. This release <% if (type !== "patch") { %>adds some new features and<% } %>fixes several bugs found in the previous release. <% if (type === "major") { %>This release also has some breaking changes, so please read the following closely. <% } %>
```
Comment (not-an-aardvark) [inline templates/blogpost.md.ejs]: Possibly in another PR, should we fix the pluralization here too to avoid things like https://github.com/eslint/eslint.github.io/pull/372?

Comment (kaicataldo) [inline templates/blogpost.md.ejs]: Yeah, let's do it in a separate PR!
